configs/test/bot/setup/android.bash

```diff
@@ -13,6 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function run_bot () {
+  bot_directory=$INSTALL_DIRECTORY/bots/$(echo $1 | sed s/:/-/)
```

Comment:
move $(echo $1 | sed s/:/-/) into a local or atleast put $1 into local and use throughout.

Reply:
Moved $1 into a local.